db/schema.rb

```diff
@@ -1270,7 +1270,7 @@
   add_foreign_key "spree_tax_rates", "spree_zones", column: "zone_id", name: "spree_tax_rates_zone_id_fk"
   add_foreign_key "spree_taxons", "spree_taxonomies", column: "taxonomy_id", name: "spree_taxons_taxonomy_id_fk"
   add_foreign_key "spree_taxons", "spree_taxons", column: "parent_id", name: "spree_taxons_parent_id_fk"
-  add_foreign_key "spree_users", "spree_addresses", column: "bill_address_id", name: "spree_users_bill_address_id_fk"
+  add_foreign_key "spree_users", "spree_addresses", column: "bill_address_id", name: "spree_users_bill_address_id_fk", on_delete: :cascade
```

Comment:
:skull: :warning: Careful! I think that this would delete all user records that have a bill address without phone. And you didn't add a migration for this which means that it would actually not be applied to existing databases.

I don't think that we should add database logic just to make a migration work if we don't need it for the rest of the app. You can nullify the foreign keys within the migration.